docs/DeveloperGuide.adoc

```diff
@@ -837,6 +837,125 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Delete tag
+
+*MSS*
+
+1.  User requests to delete tag
+2.  AddressBook shows a list of persons
+3.  User requests to delete a specific tag in the list
```

Comment:
if it is a tag in the list, should 2. above show a list of persons or a list of tags?

Reply:
Right it should be tag, thanks!